config/devices/config.go

```diff
@@ -131,4 +134,48 @@ func Configure(p *config.Provider) {
 		r.ExternalName.GetIDFn = common.GetFullyQualifiedIDFn("Microsoft.Devices", "IotHubs", "iothub_name",
 			"IotHubKeys", "name")
 	})
+
+	p.AddResourceConfigurator("azurerm_iothub_endpoint_storage_container", func(r *config.Resource) {
+		r.Version = common.VersionV1Alpha2
+		r.References = config.References{
+			"iothub_name": config.Reference{
```

Comment:
Looks like `iothub_name` argument is [deprecated](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/iothub_endpoint_storage_container#iothub_name) in favor of `iothub_id`. We can add a referencer for the `iothub_id` argument using [`rconfig.ExtractResourceID`](https://github.com/crossplane-contrib/provider-jet-azure/blob/ebac3f7f2f4d8b9e89e255d0d76c7d703df4cd1f/apis/rconfig/config.go#L51) as the extractor.

Reply:
Let's not risk the planned delivery with a native provider version bump then. Thank you @ytsarev for checking it.